templates/RELEASE_NOTES_unified.md.tmpl

```diff
@@ -19,6 +21,33 @@ to their releases:
 
 ## Upgrade Instructions
 
+### Conjur
```

Comment:
```suggestion
### Installation Instructions for the Suite Release Version of Conjur
```
what do you think about this as an alternate title? then it differentiates between upgrade instructions, which I'm not sure is exactly what we're providing here

eg if I already have a docker-compose system running, I'm not going to follow this to _upgrade_ - it would just allow me to deploy another conjur cluster. no data is transferred, I just start fresh. even so, it doesn't tell me how to destroy my old cluster (if I'd even want to just start fresh). in sum, these are _install_ instructions, not _upgrade_ instructions (which is fine)

Reply:
That's reasonable. I've gone ahead and added an install instructions section